back-end/migration.js

```diff
@@ -1,4 +1,4 @@
-require('dotenv').config()
+require('dotenv').config({ path: `${__dirname}/.env.prod` })
```

Comment:
@amelieykw it should be the default `.env` just the `.env` file should have different content